Kernel/Graphics/Intel/Plane/G33DisplayPlane.cpp

```diff
@@ -21,15 +21,16 @@ IntelG33DisplayPlane::IntelG33DisplayPlane(Memory::TypedMapping<volatile PlaneRe
 {
 }
 
-ErrorOr<void> IntelG33DisplayPlane::set_plane_settings(Badge<IntelDisplayConnectorGroup>, IntelGraphics::GPUVirtualAddress aperture_start, PipeSelect pipe_select, size_t horizontal_active_pixels_count)
+ErrorOr<void> IntelG33DisplayPlane::enable(Badge<IntelDisplayConnectorGroup>)
```

Comment:
Again, can we squash ` Kernel/IntelGraphics: Move DisplayPlane enable code to derived classes ` into the commit that introduces `G33DisplayPlane` ?